unattended.sh

```diff
@@ -114,10 +114,19 @@ print_step_end
 #
 
 print_step_begin "PHP configuration"
-timezone=`date '+%Z'`
-if [ -z "$timezone" ] ; then
-  timezone=UTC
-fi
+timezone=`php -r '
+    $timezoneName = timezone_name_from_abbr(trim(shell_exec("date \"+%Z\"")));
+
+    if (preg_match("/Time zone: (\S+)/", shell_exec("timedatectl"), $matches)) {
+        $timezoneName = $matches[1];
+    }
+
+    if (date_default_timezone_set($timezoneName) === false) {
+      $timezoneName = "UTC";
+    }
+
+    echo $timezoneName;
+' 2> /dev/null`
 echo "date.timezone = $timezone" > /etc/opt/rh/rh-php72/php.d/10-centreon.ini
```

Comment:
Caution : this file may not exists : 
https://docs.centreon.com/current/en/upgrade/upgrade-from-19-04.html#update-the-php-version

Reply:
I think that this script is done to make an install from scratch.
It should not be executed on a machine that already has a Centreon installed.

If this requirement is not met, many things could fail while executing this script.